{"filename": "lld-15.0.7-2-x86_64.pkg.tar.zst", "name": "lld", "base": "lld", "version": "15.0.7-2", "desc": "Linker from the LLVM project", "csize": "1909973", "isize": "6948647", "md5sum": "9c1a092c59d7c2dcc6b1b441a5fbbfde", "sha256sum": "124c565bb25a2774e31cf204017b86f4de3277ea4ff6f8aa7e9900bdcda7039c", "pgpsig": "iQEzBAABCAAdFiEEhs/8qRjPOvRxR1iAUeixSKmZnDQFAmQCOAIACgkQUeixSKmZnDTJegf9HLH5dVMfpgfzbb4Kark4MWFbFSeotB8kTESjSi2Db8+YNiLcuJpmAmsFgJNf4uS6xjcCyIJxmHAZ54Z35OaBBwnAmTgViET8U9/ZdJDmHfzD59X57IRuCz2NZn6aJpZTQg/0b0hZrjwRo1N0IB5jLns7LQiUmUOCjWIly3T86Di9QyTNeJBMPdEnSNmbfD0S2vTWfTLLlMRzgQ1RfTDp4hrML4s1/yXv3jGHHFxSBHRn34sSNSzypHaXWzZL2rfhPBXOZ5xq8Oht/MArFcpZ8g4MqlOO9Ia1k7tTpTFmPTpIuU1CjWylBZRTQB7uwzwEszyr3XSlWepjDE8eFAQtDg==", "url": "https://lld.llvm.org/", "license": "custom:Apache 2.0 with LLVM Exception", "arch": "x86_64", "builddate": "1677865446", "packager": "Evangelos Foutras <foutrelis@archlinux.org>", "depends": ["llvm-libs"], "makedepends": ["llvm", "cmake", "ninja", "python-sphinx"], "tokens": ["lld", "lld_x86_64", "lld_testing", "lld_x86_64_testing"]}